wkcuber/api/MagDataset.py

```diff
@@ -165,11 +171,16 @@ def __init__(
         block_len: int,
         file_len: int,
         block_type: int,
+        exists: bool = True,
```

Comment:
Personally, I'd rename this (and invert the semantics) to `create` to signal the action this will have. But I don't know where this is coming from. Maybe there is a stronger reason for `exists`?